### PR TITLE
Main page

### PR DESCRIPTION
**Pull request by gscarv13**
This is the second project in Microverse's HTML and CSS module. It consists of a reproduction of TheNextWeb homepage. It contains all the relevant content from the original page.

Built using HTML and CSS only.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,52 @@
...
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> HTML & CSS 2.0 - Project 2: Building with Responsive Design
+
+![screenshot](./assets/Screenshot-TNW.jpg)
+
+This is a project in Microverse's HTML and CSS module. It consists of a reproduction of [TheNextWeb](https://thenextweb.com/) homepage. Advertisements and many of the interactive elements have been left out.
```
Comment (arslanbisharat) [inline README.md]: - [x] Please remove the part **Advertisements and many of the interactive elements have been left out.** and add about the things you did in the project.

File: README.md [hunk 2]
```diff
@@ -1 +1,52 @@
...
+
+## Live Demo
+
+[HTML & CSS 2.0 - Building with Responsive Design](https://jpdf00.github.io/Html-and-CSS-2.0-Project-2-Building-with-Responsive-Design/)
+
+
+## Getting Started
+
+Visit the live version on the link above.
+
```
Comment (arslanbisharat) [inline README.md:24]: - [x] Instead of this please add instructions to clone your project and make it work on the local host

File: README.md [hunk 3]
```diff
@@ -1 +1,52 @@
...
+Feel free to check the [issues page](https://github.com/jpdf00/Html-and-CSS-2.0-Project-2-Building-with-Responsive-Design/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](lic.url) licensed.
```
Comment (arslanbisharat) [inline README.md]: - [x] Please add a path to the license.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Request changes 😳

I am very happy to see this project and I can see you worked hard on this project. There are only a few issues that we need to address and I am sure that you will end up with a perfect project

_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_

This is a great opportunity to learn. Microverse is here to help you. 

**Please have a look at the suggestions to get more understanding.**

- [x] Please remove linter errors from your project. 

- [x] Please add semantic tags to your code. You are missing the header tag. 

- [x] Please add the proper path to the images and other files. 
```

// Bad Paths
<link rel="stylesheet" href="/assets/stylesheets/main.css">
<link rel="stylesheet" href="/assets/js/bootstrap.min.js">
<img src="/imgs/myimage.png">


// Good Paths
<link rel="stylesheet" href="./assets/stylesheets/main.css">
<link rel="stylesheet" href="./assets/js/bootstrap.min.js">
<img src="./imgs/myimage.png">

```

- [x] [Optional] Please add a small description of your project in the about section of this repo. 
This will help you to present your project effectively. 
- [x] [Optional] Please add CSS and images folder in a single folder. This will help you to make your project cleaner. 

- [x] This section looks amazing but please move this section a little right. 
chrome-capture (17)



**Please make the changes and request a re-review for the given project**

If you get stuck somewhere, don't hesitate to reach out or schedule s 1:1 meeting with a Full-time TSE. 

Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Review by layzrd — CHANGES_REQUESTED** [review]
### Status: Require changes 🌳 ♲
Hi @, I am [Abdulaziz Redi Ali](https://azizali.ml/), your TSE for this PR.  👏👏  I appreciate your hard work in replicating thenextweb.com. Most requirements have been done 👌,  kindly fix the following issues.

[REQUIRED]

On small screens, the search icon and the hamburger menu should be on the rightmost side of the header row.
	Use `min-width `around `768px` for medium screens

|Your Copy | Source |
| :---:         |     :---:      |
|Screen Shot 2020-12-03 at 6 17 26 PM|Screen Shot 2020-12-03 at 6 27 34 PM|

Once you fix the issues, kindly submit another code review. ♲

Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/) | [Contact me on Slack](https://microverse-students.slack.com/team/U011SD6JUH2)

**Review by descholar-ceo — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
## :clap: :clap: :clap: Congratulations @jpdf00 and @gscarv13 , your project is approved :tada: :tada: :tada: 
You have implemented all minimum requirements for this project to mark it as complete, it doesn't need more changes, you can go ahead and merge all of the changes on base branch,

Don't forget to fill the mandatory project completion form, and see you to next project,

Happy coding :computer: :computer: :computer: 

Re-reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Pull request merged**